packages/widgets/src/renderwidget.ts

```diff
@@ -0,0 +1,225 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2017, PhosphorJS Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+|
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
+import {
+  Message, MessageLoop
+} from '@phosphor/messaging';
+
+import {
+  ISignal
+} from '@phosphor/signaling';
+
+import {
+  VirtualContent, VirtualDOM
+} from '@phosphor/virtualdom';
+
+import {
+  Widget
+} from './widget';
+
+
+/**
+ * An object which can be used as a model for a render widget.
+ */
+export
+interface IRenderModel {
```

Comment:
Nice, this was the main point I wanted to make - that this should be a simple interface with only the signal. This will allow us to use it with our Redux style state stores as well. Is the name of the signal the same in both places?